CommerceCloud/Integration/03_SalesforceParticularities.md

```diff
@@ -0,0 +1,170 @@
+[!!Manage the Salesforce connection](./02_ManageSalesforceConnection.md)
+[!!Manage the ETL mappings in DataHub](../../DataHub/Operation/01_ManageETLMappings.md)
+[!!User interface Connections](../UserInterface/01_Connections.md)
+
+# Salesforce product data particularities
+
+There are several particularities regarding the Salesforce product data that must be taking into account when establishing and using the Salesforce connection. These particularities are described in detail below.  
+
+## Master catalog
+
+A catalog is a product tree. You can assign certain nodes from a certain catalog to a product and thus divide it into different categories. The product data are part of the catalog. Every product in Salesforce must belong to a catalog. This catalog, who owns the product, is the master catalog. The product belongs to the master catalog but must not be assign to a category within the master catalog. The master catalog in Salesforce determines which fields are available for a product.   
+Regardless of which catalog a product belongs to, the product can be assigned to other catalogs as well.
+
+The master catalog who owns the products must be mapped in the *DataHub* module to the *Master Catalog (CommerceCloud Actind)* destination attribute.
+The ETL mapping determines which catalog is used as master catalog in Salesforce.   
+It is mandatory that the master catalog is set in all ETL mappings to the Salesforce attribute set, see [Configure the master catalog](./02_ManageSalesforceConnection.md#configure-the-master-catalog).
+
+In Actindo, all fields from Salesforce are available, as there exists only one single attribute set for Salesforce.    
+
+
+## Sites
+
+Salesforce offers the possibility to create different sites for the order import. In Actindo, these sites are created as sub-shops. That means that every single shop is a sub-account, available in the stock withdrawal matrix in the *Warehouse* module.
+
+The different sites are managed in Salesforce: *Administration > Sites > Manage Sites*
+
+![Manage sites](../../Assets/Screenshots/Channels/Settings/Connections/CommerceCloud/ManageSites.png "[Manage sites]")
+
+
+## Attributes
+
+In Salesforce, an attribute can either be *Localizable*, *Site-Specific* or neither. Also, an attribute can only be either *Localizable* or *Site-Specific*, not both at the same time. A localizable attribute in Salesforce is equivalent to a multi-language attribute in Actindo. A site-specific attribute in Salesforce is similar but not the same as a multi-channel attribute in Actindo. 
+
+Actindo supports both localizable attributes and site-specific attributes from Salesforce. However, Actindo does not support site-specific boolean attributes from Salesforce because they have a different number of states in Actindo and Salesforce, which would cause problems. 
+
+The localizable attributes from Salesforce are imported to Actindo as multi-language attributes.
```

Comment:
Here is missing that the languages in actindo and commercelcoud are mapped via the language key

Reply:
ich seh grad es kommt unten noch aber trotzdem wäre es hier gut zu erwähnen glaube ich